htmlelements-java/src/main/java/ru/yandex/qatools/htmlelements/annotations/Timeout.java

```diff
@@ -0,0 +1,12 @@
+package ru.yandex.qatools.htmlelements.annotations;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Retention(RetentionPolicy.RUNTIME)
+@Target({ElementType.TYPE, ElementType.FIELD})
+public @interface Timeout {
```

Comment:
take a look at [test for AjaxElementLocator](https://github.com/SeleniumHQ/selenium/blob/master/java/client/test/org/openqa/selenium/support/pagefactory/AjaxElementLocatorTest.java) in selenium codebase